config/clusters/qcl/common.values.yaml

```diff
@@ -54,3 +54,142 @@ jupyterhub:
           - quantifiedcarbon:JupyterHub
         scope:
           - read:org
+  singleuser:
+    image:
+      # User image repo: https://quay.io/repository/linkedearth/pyleoclim
+      name: pangeo/pangeo-notebook
+      # pullPolicy set to "Always" because we use the changing over time tag
+      # "latest".
+      pullPolicy: Always
+      tag: "latest"
+    profileList:
+      # NOTE: About node sharing
+      #
+      #       CPU/Memory requests/limits are actively considered still. This
+      #       profile list is setup to involve node sharing as considered in
+      #       https://github.com/2i2c-org/infrastructure/issues/2121.
+      #
+      #       - Memory requests are different from the description, based on:
+      #         whats found to remain allocate in k8s, subtracting 1GiB
+      #         overhead for misc system pods, and transitioning from GB in
+      #         description to GiB in mem_guarantee
+      #         https://cloud.google.com/kubernetes-engine/docs/concepts/plan-node-sizes.
```

Comment:
Yikes I struggle to understand my own comment above.

I disregarded the steps described in this linked documentation though when I was trying to understand how much memory was available for pods in k8s. I don't think there is an algorithm documented to take us from the reported specification of 128 GB to the allocatable memory as the linked documentation provides among other things.

So in a way, this was not used for the small/medium memory calculations - only the directly reported value for allocatable memory of a node of given types. I got these values using `kubectl get node | grep -A20 allocatable`

I think it could make sense to relocate this link to its own paragraph for readability, but I'd also want to rework this entire comment for all hubs with node sharing in a standalone PR.

No change requested, feel free to mark as resolved!

Reply:
Yes. I think a clearer writeup in docs of shared node profile configuration would be helpful but yes lets not block QCL turnup on that.